### DISCUSSION TITLE
Why does YOLOv8 not use ResNet as a backbone?

### DISCUSSION THREAD

**Discussion by Matt-Kou** [ultralytics/ultralytics#505] [Q&A]
Can we use resnet as a backbone? if so, why the architectures of YOLOv8 don't use it? Is it because ResNet is slower? Or the MAp would be lower?

**Comment by glenn-jocher** [comment]
@Matt-Kou hello there! YOLOv8 does not use ResNet as a backbone by default because we've designed it with a focus on achieving an optimal balance between speed and accuracy. The architecture choices in YOLOv8 are the result of extensive research and experimentation to ensure that the models are fast and accurate for real-time applications.

While ResNet is a powerful and widely-used backbone, it may not always provide the best speed-to-accuracy trade-off for the specific use cases YOLOv8 targets. That said, it's certainly possible to use ResNet as a backbone for YOLO models, but it's not included in the default configurations of YOLOv8.

If you're interested in experimenting with different backbones, you can certainly modify the model architecture to include ResNet. However, keep in mind that this might affect the model's performance and speed, and you would need to train the model to see how it compares to the default YOLOv8 backbones.

For more information on customizing model architectures and understanding the design choices behind YOLOv8, you can refer to our documentation. 😊🚀

**Comment by furycurry** [comment]
Hi @Matt-Kou . Were you able to integrate ResNet as a backbone in yolov8? If so, would you mind telling the steps on how to do so?
